test/unit/ObjectMakrTests/processProperties.js

```diff
@@ -1,17 +0,0 @@
-define(["mocks"], function (mocks) {
-    return function () {
-        var expect = require("chai").expect;
-
-        it("parses elements into object types", function () {
```

Comment:
It didn't make much sense to leave this in; all the function does is call to `processPropertyArray` a few times.

This is why I normally don't normally advocate for testing privates in unit tests. ObjectMakr happens to be difficult to test without delving into internals.